resources/assets/js/components/UnclaimedJobCard.tsx

```diff
@@ -92,13 +84,7 @@ const UnclaimedJobCard: React.FunctionComponent<UnclaimedJobCardProps> = ({
                   description="Header before list of HR advisors."
                   defaultMessage="HR Advisors: "
                 />
-                {hrAdvisors.map((advisor, index): string => {
-                  const comma =
-                    hrAdvisors.length !== 1 && index + 1 !== hrAdvisors.length
-                      ? ","
-                      : " ";
-                  return `${advisor}${comma} `;
-                })}
+                {hrAdvisors.join(",")}
```

Comment:
You probably want to add a space after the comma, `.join(", ")`

Reply:
Yup :p